bigframes/_config/compute_options.py

```diff
@@ -66,13 +66,19 @@ class ComputeOptions:
             engine to handle. However this comes at the cost of increase cost and latency.
         extra_query_labels (Dict[str, Any], Options):
             Stores additional custom labels for query configuration.
+        sem_ops_confirmation_threshold (int, Options):
```

Comment:
Do you mean "optional"? I'm not sure what "options" means in this context.

Also, let's spell out `semantic`. I don't think `sem` is a common enough abbreviation to be self documenting.

```suggestion
        semantic_ops_confirmation_threshold (int, optional):
```

Also, consider adding an option to fail instead of asking for confirmation. When someone automates their script, they won't want it to hang waiting for input from standard in.

Reply:
I added a "semantic_ops_threshold_autofail" option